.github/workflows/release-build-sign-upload.yml

```diff
@@ -905,20 +905,29 @@ jobs:
     needs:
     - setup
     - build-linux
-    runs-on: ubuntu-20.04
-    container:
-      image: ubuntu:20.04
+    runs-on: ubuntu-latest
     steps:
 
     - name: Download Signed Linux Packages
       uses: actions/download-artifact@v4
       with:
         name: cf-cli-linux-deb-packages
 
-    - name: Display structure of downloaded files
+    - name: Display structure of downloaded files and test compression
       run: |
         ls -R
+
+        for i in debian:11 debian:12 ubuntu:20.04 ubuntu:22.04 ubuntu; do docker run \
```

Comment:
*question* I dont have context on the compression issue, but how does this test it?

Reply:
`dpkg --info` fails when trying to open `.deb` file with unsupported compression

Reply:
👁️  https://github.com/cloudfoundry/cli/pull/3450#discussion_r1989991255

Reply:
example https://github.com/a-b/cli/actions/runs/13781247233/job/38539815075#step:3:30

Reply:
Can we use the matrix strategy here to change the container image? instead of this for loop?

Reply:
Good call. Yes, we should use matrix!